frontend/src/intl/field-labels.json

```diff
@@ -1,664 +1,695 @@
-            "nn": "Status"
-        },
-        "regulation_direction": {
-            "en": "Regulation direction",
-            "nb": "Reguleringsretning",
-            "nn": "Reguleringsretning"
-        },
-        "maximum_available_capacity": {
-            "en": "Maximum available capacity",
-            "nb": "Maksimal tilgjengelig kapasitet",
-            "nn": "Maksimal tilgjengeleg kapasitet"
-        },
-        "is_small": {
-            "en": "Small?",
-            "nb": "Liten?",
-            "nn": "Liten?"
-        },
-        "minimum_duration": {
-            "en": "Minimum duration",
-            "nb": "Minimumsvarighet",
-            "nn": "Minimumsvarigheit"
-        },
-        "maximum_duration": {
-            "en": "Maximum duration",
-            "nb": "Maksimumsvarighet",
-            "nn": "Maksimumsvarigheit"
-        },
-        "recovery_duration": {
-            "en": "Recovery duration",
-            "nb": "Gjenopprettingsvarighet",
-            "nn": "Gjenopprettingsvarigheit"
-        },
-        "ramp_rate": {
-            "en": "Ramp rate",
-            "nb": "Effektendringshastighet",
-            "nn": "Effektendringshastigheit"
-        },
-        "accounting_point_id": {
-            "en": "Accounting point ID",
-            "nb": "Avregningspunkt-ID",
-            "nn": "Avregningspunkt-ID"
-        },
-        "grid_node_id": {
-            "en": "Grid node ID",
-            "nb": "Nettnode-ID",
-            "nn": "Nettnode-ID"
-        },
-        "grid_validation_status": {
-            "en": "Grid validation status",
-            "nb": "Nettvalideringsstatus",
-            "nn": "Nettvalideringsstatus"
-        },
-        "grid_validation_notes": {
-            "en": "Grid validation notes",
-            "nb": "Nettvalideringsnotater",
-            "nn": "Nettvalideringsnotat"
-        },
-        "validated_at": {
-            "en": "Validated at",
-            "nb": "Validert ved",
-            "nn": "Validert ved"
+    "en": {
+        "controllable_unit": {
+            "id": "ID",
+            "business_id": "Business ID",
+            "name": "Name",
+            "start_date": "Start date",
+            "status": "Status",
+            "regulation_direction": "Regulation direction",
+            "maximum_available_capacity": "Maximum available capacity",
+            "is_small": "Small",
+            "minimum_duration": "Minimum duration",
+            "maximum_duration": "Maximum duration",
+            "recovery_duration": "Recovery duration",
+            "ramp_rate": "Ramp rate",
+            "accounting_point_id": "Accounting point ID",
+            "grid_node_id": "Grid node ID",
+            "grid_validation_status": "Grid validation status",
+            "grid_validation_notes": "Grid validation notes",
+            "validated_at": "Validated at",
+            "recorded_at": "Recorded at",
+            "recorded_by": "Recorded by"
+        },
+        "controllable_unit_suspension": {
```

Comment:
I would prefer if the dicionary was completly flat, so the keys are "controllable_unit_suspension.id"  since its easier to infer the key-type.